### DISCUSSION TITLE
Any way to manually add body with the PHPDocs

### DISCUSSION THREAD

**Discussion by eugene-mednefits** [dedoc/scramble#629] [Q&A]
I have a unique situation for our API Gateway

```
    public function update(Request $request): JsonResponse
    {
        $data = $request->all();
        return $this->response($data);
    }
```
    
We are only passing the data through, but we would like to build a body for the request. Is there any way to achieve this?

**Comment by shavonn** [comment]
Bottom of the page https://scramble.dedoc.co/usage/response
